lib/active_set/sorting/operation.rb

```diff
@@ -0,0 +1,165 @@
+# frozen_string_literal: true
+
+require_relative '../attribute_instruction'
+
+class ActiveSet
+  module Sorting
+    class Operation
+      def initialize(set, instructions_hash)
+        @set = set
+        @instructions_hash = instructions_hash
```

Comment:
A valid `instructions_hash` would look like:
```
{
  AttributePathString: DirectionStringable
  Stringable: {
    AttributePathString: DirectionStringable
  }
}
```

Much like filtering, the instructions hash for sorting works with attribute instructions, which can either be dot-separated or used nested hashes. Unlike filtering, values must be a direction (like `:desc` or 'ASC' or similar)